packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -133,9 +132,10 @@ export const prepareStatusManager = (
       storedCompletedTxs.add(txId);
     }
 
-    const capData = await E(marshaller).toCapData(record);
-
-    await E(txNode).setValue(JSON.stringify(capData));
+    // Don't await, just writing to vstorage.
+    void E.when(E(marshaller).toCapData(record), capData =>
+      E(txNode).setValue(JSON.stringify(capData)),
```

Comment:
I wonder if we should be using...

```suggestion
      E.sendOnly(txNode).setValue(JSON.stringify(capData)),
```

I don't see any other use of it in `agoric-sdk` production code, though.